packages/taquito-michelson-encoder/src/tokens/comparable/timestamp.ts

```diff
@@ -12,7 +12,10 @@ export class TimestampToken extends ComparableToken {
   }
 
   public Execute(val: { string?: string; int?: string }) {
-    if (val.string) {
+    if (val.string && /^\d{10}(\.\d)?$/.test(val.string)) {
```

Comment:
If we have a Unix timestamp that refers to the 2000s or prior, the regex will fail as the number of digits will be < 10.
Should the check be less restrictive?

Reply:
Good point, I can make it less restrictive. Do you think 9-10 digits is enough? or should we be making a larger range like 8-10?

Reply:
What do you think about just validating that the string is made of digits regardless of the length?

Reply:
we can definitely do that too, I can make that change then